Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -116,6 +129,43 @@ public class TextAttachment: NSTextAttachment
 
         return CGRect(origin: CGPointZero, size: CGSize(width: width, height: onScreenHeight(width)))
     }
+
+    func updateImage() {
+        switch kind {
+        case .RemoteImage:
+            break
+        default:
+            return
+        }
+        guard !isFetchingImage else {
```

Comment:
I'd consider adding `let imageProvider = imageProvider` to this guard, since none of the code below makes sense without a delegate.